src/components/DocumentationTopic/PrimaryContent/DeclarationDiff.vue

```diff
@@ -70,6 +70,8 @@ export default {
 @import 'docc-render/styles/_core.scss';
 
 .declaration-diff {
+  background: var(--background, var(--color-code-background));
```

Comment:
We should do this in the `DeclarationGroup` component as well.

Reply:
No need. This is just for the diff.

Reply:
I thought we need the gray background for `declarations` when API Changes is shown as well.

Reply:
Noted.